Stellar-ledger.x

```diff
@@ -589,9 +589,9 @@ struct LedgerCloseMetaV1
     // other misc information attached to the ledger close
     SCPHistoryEntry scpInfo<>;
 
-    // Size in bytes of BucketList, to support downstream
+    // Size in bytes of live Soroban state, to support downstream
     // systems calculating storage fees correctly.
-    uint64 totalByteSizeOfBucketList;
+    uint64 totalByteSizeOfLiveSorobanState;
```

Comment:
This has to be rebased and the change should also go to meta v2.

Reply:
Hmm the git diff display was kinda funky, we already have the rename in v1. Rebased, should be fine now.